includes/compatibility/edd/filters.php

```diff
@@ -30,11 +30,28 @@ function themedd_edd_settings_styles( $settings ) {
 	unset( $settings['main']['disable_styles'] );
 
 	// Remove "Default Button Color" option since Themedd controls all button styling
+	unset( $settings['button_text']['checkout_color'] );
 	unset( $settings['main']['checkout_color'] );
 
+	// Remove the button style selector.
+	unset( $settings['button_text']['button_style'] );
+
 	return $settings;
 }
-add_filter( 'edd_settings_styles', 'themedd_edd_settings_styles' );
+add_filter( 'edd_settings_misc', 'themedd_edd_settings_styles' );
+
+add_action( 'admin_menu', 'themedd_filter_edd_settings' );
+/**
+ * For EDD 2.x, use the `edd_settings_styles` filter to remove style settings.
+ *
+ * @return void
+ */
+function themedd_filter_edd_settings() {
+	if ( version_compare( '2.10.999', EDD_VERSION, '>' ) ) {
+		return;
+	}
+	add_filter( 'edd_settings_styles', 'themedd_edd_settings_styles' );
+}
```

Comment:
This isn't working as is (probably too late of a hook). But this will work instead of the function:

```php
if ( defined( 'EDD_VERSION' ) && version_compare( '2.10.999', EDD_VERSION, '>' ) ) {
	add_filter( 'edd_settings_styles', 'themedd_edd_settings_styles' );
}
```